### DISCUSSION TITLE
Printing Wall Information

### DISCUSSION THREAD

**Discussion by JhonCordova** [AMReX-Combustion/PeleC#860] [Q&A]
Hi everyone,

I would like to print wall shear stress or velocity gradients data at the walls. Is it possible to have this information within PeleC?

Thanks in advance!

**Comment by baperry2** [comment]
There isn't a built in way to do this. You could post-process it fairly easily from velocity and viscosity data in a plt file. You could also add a bit of code to print out the appropriate component of `flx` at the boundary around here: https://github.com/AMReX-Combustion/PeleC/blob/b9ebe80bd3b00db0f50710048e50bac01212ef2b/Source/Diffusion.cpp#L329
